README.md

```diff
@@ -53,7 +54,7 @@ docker-compose version
 docker run hello-world
 ```
 
-On Windows, these commands must be run inside a WSL 2 terminal.
+> :warning: **On Windows, all commands must be run inside a WSL 2 terminal.**
```

Comment:
Warning?

Reply:
Dan was confused if things should be run in Power Shell or WSL 2 terminal on windows.